content/ko/docs/helm/helm_show.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Helm Show"
+---
+
+## helm show
+
+차트의 정보를 보여준다
+
+### 개요
+
+
+이 명령어는 차트에 대한 정보를 표시하는 여러 하위 명령어들로 구성된다
+
+
+### 옵션
+
+```
+  -h, --help   show 에 대한 도움말
+```
+
+### 부모 명령어에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  파일처럼 표준 오류로도 로그 출력
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```

Comment:
Why is this default changing from what's in the English docs?

Reply:
I will modify every I PRed as your comment like issue #810 :D